client/ayon_blender/api/capture.py

```diff
@@ -133,7 +134,22 @@ def isolate_objects(window, objects):
 
     deselect_all()
 
+def restore_global_view(window, isolate=None):
```

Comment:
Why would we care about the `isolate` behavior? I'd remove that - which simplifies the function quite a bit.

Also, I really have a feeling that somehow this should be part of the independent window context manager - but no idea whether that makes any sense from Blender API point of view.

Reply:
You are right! It does make more sense to call the function inside independet_window - right before closing it!